pkg/operation/botanist/component/resourcemanager/resource_manager_test.go

```diff
@@ -421,9 +423,6 @@ var _ = Describe("ResourceManager", func() {
 					ProjectedTokenMount: resourcemanagerv1alpha1.ProjectedTokenMountWebhookConfig{
 						Enabled: true,
 					},
-					SeccompProfile: resourcemanagerv1alpha1.SeccompProfileWebhookConfig{
-						Enabled: true,
-					},
```

Comment:
Why did you remove this block? It does not really relate to the topology-aware routing.
Was this webhook removed?

Reply:
There isn't a functional change. Previously in the test code the seccomp profile webhook was enabled by default and explicitly disabled when targetKubeconfig != nil (target cluster != source cluster aka GRM part of the Shoot control plane). With this change the seccomp profile webhook is disabled by default and enabled when targetKubeconfig == nil (target cluster = source cluster aka Seed GRM).
I made this change for consistency with other Seed GRM webhooks like CRDDeletionProtection webhook and ExtensionValidation webhook.